### DISCUSSION TITLE
How about offline accounts?

### DISCUSSION THREAD

**Discussion by hidetoshi04** [mircokroon/minecraft-world-downloader#165] [Q&A]
Hello we have an offline server in which I only use my offline account. Do you provide options for that? I hope you do.

**Comment by mircokroon** [comment]
Offline mode works, yes. No special configuration is required.

**Reply by hidetoshi04** [reply]
but it asks for a mojang account right?

**Reply by mircokroon** [reply]
Yes but if you're running in offline mode you can ignore this I believe, does it refuse to connect for you?

**Reply by hidetoshi04** [reply]
it tells that I need authentication or something. If you can't replicate it, I will send some screenies for you when I'm already home

**Reply by mircokroon** [reply]
I tried offline accounts before and I didn't run into any issues, some screenshots would be appreciated yeah. 🙂
